test/extended/operators/olm.go

```diff
@@ -70,4 +71,18 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			}
 		})
 	}
+
+	// OCP-24061 - [bz 1685230] OLM operator should use imagePullPolicy: IfNotPresent
+	g.It("have imagePullPolicy:IfNotPresent on thier deployments", func() {
```

Comment:
Actually, the same `imagePullPolicy` setting for all of the 2nd(Managed by CVO) operators. It would be great if you can automate it for all other components in one automation case. Anyway, for this automation case, it looks good to me.

Reply:
Thanks, I will think more generic in the next PRs